components/ActionSheet/examples/ActionSheetComparisonExampleViewController.m

```diff
@@ -46,15 +46,7 @@ - (instancetype)init {
 - (void)viewDidLoad {
   [super viewDidLoad];
 
-  id<MDCColorScheming> colorScheme;
```

Comment:
If this code was generating a compiler error, then I think it's good to change it. If not, I think it should be moved to a separate PR and this should focus only on the critical path to make changes to MDCContainerScheming.

Reply:
Removed